scripts/babel/defaultLayout.js

```diff
@@ -1,17 +1,23 @@
 const path = require("path");
-const fs = require("fs");
+const findUp = require("find-up");
```

Comment:
Is `scripts/babel` the best location for whats essentially "backend" code? Following Artsy patterns scripts handles things like start, deploy, etc, vs internal stuff.

Reply:
Perhaps not. I could put `babel` at the root? I didn't put much thought into that decision other than trying to keep the root as clean as I reasonably could.

Reply:
How about `lib`? That seems sufficiently internal and tidy

Reply:
Also, would it be possible to move sourcecode into an `src` folder? I know that `next` supports that.

Reply:
We can move everything _but_ pages. It's gotta be root. I'd almost prefer to keep things at the root, just have them simple.

Reply:
Np 👍